doc/polyglossia.tex

```diff
@@ -724,42 +787,44 @@ \subsection{afrikaans}\label{afrikaans}
 	\end{shorthands}
 \end{itemize}
 
+%\subsection{amharic}\label{amharic}
+
```

Comment:
Why is this re-added?

Reply:
I don't know why *amharic* is present here. I just corrected the alphabetical ordering. So far, it occurred before *afrikaans*.

Reply:
Please remove it.

Reply:
OK.